ipn/ipnlocal/local.go

```diff
@@ -4085,3 +3986,26 @@ func (b *LocalBackend) SetDevStateStore(key, value string) error {
 func (b *LocalBackend) ShouldInterceptTCPPort(port uint16) bool {
 	return b.shouldInterceptTCPPortAtomic.Load()(port)
 }
+
+func (b *LocalBackend) SwitchProfile(profile string) error {
+	if err := b.pm.SwitchProfile(profile); err != nil {
+		return nil
```

Comment:
Should this be `return err`? I had a bug where I was passing in the profile name instead of the ID, and the "not found" error was being dropped.

Also, may want to add an explicit `ProfileID` type instead of `string` so that it's more obvious how profiles are identified.

Reply:
ah yes it should be return err.

I was thinking about making the ProfileID a type too. Will do.